src/main/java/org/isf/patient/model/Patient.java

```diff
@@ -506,6 +512,26 @@ public void setDeleted(char deleted) {
 
 	public PatientProfilePhoto getPatientProfilePhoto() {
 		return patientProfilePhoto;
+	}	
+
+	public String getAnamnesis() {
+		return anamnesis;
+	}
+
+	public void setAnamnesis(String anamnesis) {
+		this.anamnesis = anamnesis;
+	}
+
+	public String getAllergies() {
+		return allergies;
+	}
+
+	public void setAllergies(String allergies) {
+		this.allergies = allergies;
+	}
+
+	public void setName(String name) {
+		this.name = name;
 	}
```

Comment:
@ArnaudFonzam do you really need this setter in "ui"? the field `name` is meant as full name and it's automatically set with setFirstName and setSecondName

Reply:
ok, I will remove for now because seems not used if not in tests only.

Reply:
> ok, I will remove for now because seems not used if not in tests only.

as not said, we'll keep as POJO standard (was the only setter missing) but I will modify it to make the change consistent

Reply:
ok if it is automatically set we can remove it